src/output.cpp

```diff
@@ -1958,8 +1958,9 @@ void play_music(std::string)
 {
 }
 
-void play_sound(std::string)
+void play_sound_effect(std::string id, std::string variant, int volume)
 {
+    (void)id;(void)variant;(void)volume;
```

Comment:
> However, the parameters are actually used in the alternative branch.

In the other implementation of that function, yes (the current code is fine there). But not in this implementation, this implementation is only 1 line: the various `(void)x;` statements